src/batch/src/executor/monitor/stats.rs

```diff
@@ -46,12 +44,189 @@ impl BatchMetrics {
         )
         .unwrap();
 
+        // Spawn a deletor.
+        // TaskMetricsManager will create task metrics for each BatchExecution and task metrics will
+        // record all the label it used. When the BatchExecution is finished, the task metrics will
+        // send all record to the delete queue. deletor is responsible for reading
+        // unused record from delete queue periodically and remove them from metrics.
+        // The deletor will remove the record after several minutes. Because it's important not to
+        // delete it immediately since prometheus pull data periodically.
+        let (delete_queue_sender, mut delete_queue_receiver) =
+            tokio::sync::mpsc::channel::<DeleteRecord>(1024);
+        let exchange_recv_row_number_clone = exchange_recv_row_number.clone();
+        tokio::spawn(async move {
+            let mut interval = tokio::time::interval(Duration::from_secs(60));
+            let mut delete_cache: Vec<DeleteRecord> = Vec::new();
+            let mut connect = true;
+            while connect {
+                // run every minute.
+                let _ = interval.tick().await;
+
```

Comment:
add some trace here?

Reply:
+1 Please add some info log here so that we can know whether it's alive in debugging.